packages/notifi-react-example/src/App.tsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 
 import './App.css';
 import { NotifiCard } from './NotifiCard/NotifiCard';
+import { WalletConnectCard } from './NotifiCard/WalletConnectCard';
 
 function App() {
   return (
     <div className="App">
       <NotifiCard />
```

Comment:
I know in our app we have a few different cards, could we limit them through a parameter we pass down?

Or maybe we can do a toggle, I think there are several requests being sent and using a walletConnect might be adding some more.

I know we have the multi wallet too, maybe we can separate them into different views